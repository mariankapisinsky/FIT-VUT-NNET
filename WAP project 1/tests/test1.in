18,11,34,31,3,46,41,67,20,92,98