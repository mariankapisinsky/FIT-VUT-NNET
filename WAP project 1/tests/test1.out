preorder
18,11,3,34,31,20,46,41,67,92,98
inorder
3,11,18,20,31,34,41,46,67,92,98
postorder
3,11,20,31,41,98,92,67,46,34,18